website/docs/agent-installation.md

```diff
@@ -193,4 +193,4 @@ Installation Mode:  cluster
 👉 Litmus agents can be accessed here: http://172.17.0.2:31696/targets
 ```
 
-To verify, if the connection process was successful you can view the list of connected agents from the Targets section on your LitmusPortal and ensure that the connected agent is in Active State.
+To verify, if the connection process was successful you can view the list of connected agents from the targets section on your LitmusPortal and ensure that the connected agent is in Active State.
```

Comment:
@ksatchit : I'll let you advise on this nomenclature change in the documentation.